elixir-mode.el

```diff
@@ -185,7 +185,9 @@ for the Elixir programming language."
     `(
       (atoms . ,(rx ":"
                     (or
-                     (one-or-more (any "a-z" "A-Z" "_" "\"" "'"))
+                     (and
+                      (any "a-z" "A-Z" "_" "\"" "'")
+                      (zero-or-more (any "a-z" "A-Z" "0-9" "_" "\"" "'")))
```

Comment:
Why the change from `one-or-more` to `zero-or-more`? This doesn't make sense to me.